### PR TITLE
[p2p] added some p2p (potato to potato) code

### PR DESCRIPTION
**Pull request by pdlla**
-just a first draft. Mainly to test pull request
and code review D:

testing: none! Too hard.

### CODE REVIEW

File: erlang/p2p/node.erl [hunk 1]
```diff
@@ -0,0 +1,52 @@
+-module(node).
```
Comment (pdlla) [inline erlang/p2p/node.erl]: hello anton

File: erlang/p2p/node.erl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+test(Tp) ->
+    io:fwrite("hello, world\n"),
+    if Tp == 1 ->
+            init();
+       Tp == 2 ->
+            client()
+    end.
+
+client() ->
```
Comment (pdlla) [inline erlang/p2p/node.erl]: testing out "start a review" button

File: erlang/p2p/node.erl [hunk 3]
```diff
@@ -0,0 +1,52 @@
+-module(node).
+
+-export([test/1]).
+
+-include("../blocktree/blocktree.hrl").
```
Comment (antonbobkov) [inline erlang/p2p/node.erl]: not used! so bad

File: erlang/p2p/node.erl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+-include_lib("stdlib/include/assert.hrl").
+
+test(Tp) ->
+    io:fwrite("hello, world\n"),
+    if Tp == 1 ->
```
Comment (antonbobkov) [inline erlang/p2p/node.erl]: can use client and server instead of 1 and 2

File: erlang/p2p/node.erl [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    tcp_listener(LSock).
+
+handle(Socket) ->
+    io:fwrite("handling socket"),
+    %% initiate handshake
+    %% wait for handshake
+    %% go to receive handle
+    inet:setopts(Socket, [{active, true}]),
+    receive
+        {tcp, Socket, <<"quit", _/binary>>} ->
```
Comment (antonbobkov) [inline erlang/p2p/node.erl]: such binary

Comment (pdlla) [reply erlang/p2p/node.erl]: so comments

Comment (antonbobkov) [reply erlang/p2p/node.erl]: so scary

File: erlang/p2p/node.erl (hunk 1 above)
Comment (antonbobkov) [reply erlang/p2p/node.erl]: hiiiiiiiiiiiiiiiiiiiiiiiiiiii

File: erlang/p2p/node.erl (hunk 2 above)
Comment (antonbobkov) [reply erlang/p2p/node.erl]: much testing

File: erlang/p2p/node.erl (hunk 3 above)
Comment (pdlla) [reply erlang/p2p/node.erl]: D:

File: erlang/p2p/node.erl (hunk 4 above)
Comment (pdlla) [reply erlang/p2p/node.erl]: 👍

**Pull request merged**